static/docs/api/frameworks/atomcore/class_a_z_1_1_data_1_1_instance_data-members.html

```diff
@@ -20,7 +20,7 @@
   <td id="projectlogo"><img alt="Logo" src="/images/api-reference/api-ref-icon.png"/></td>
```

Comment:
In the future, it'd be nice if this logo had a clickable link to take you back to the [API Reference homepage](https://www.o3de.org/docs/api/frameworks/). As it is, there's no way to navigate to any other API other than using the browser back button. I realize that the API Reference is already scheduled for an overhaul. I just mention it in case it's a quick interim fix. Filed as a GHI: https://github.com/o3de/o3de.org/issues/2012